### DISCUSSION TITLE
What is the relation between subgroup tests?

### DISCUSSION THREAD

**Discussion by Bazni** [fzyzcjy/flutter_convenient_test#367] [Q&A]
If I have several tests in a group, does the app restart for each test? Is it the case for each sub group?

To add more context, when my second test of my group started the app stayed in a loading state. It surely comes from my code 😅
The only error I got is that the widget I'm looking for is not found.

**Comment by fzyzcjy** [comment]
You can use isolate mode, which will call `hot restart` for your app for each test, thus you get full isolation. Alternatively, you can modify your app, such that its states are properly encapsulated. I personally use the former way in my app :)

**Reply by Bazni** [reply]
I will try thank you

**Reply by fzyzcjy** [reply]
You are welcome!
